setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup, find_packages
 
-__version__ = '1.3.0'
+__version__ = '1.4.0'
```

Comment:
nit: we don't have a clear guideline currently to define our version. But I think bump it to 1.3.1 as the change is not very disruptive. WDYT?

Reply:
Agree, see https://github.com/lyft/amundsenmetadatalibrary/pull/80#discussion_r327764872

I was mostly “inspired” by the 1.3.0 bump, been wanting to have that clarification for a while, but held back on opening it since it’s not *that* important compared to - just - progressing 🙂, but now that it’s opened let’s see if we can find quick clarification! (Let’s also move talking to https://github.com/lyft/amundsenmetadatalibrary/pull/80)